OSDOCS-2557: Add the overview page for registry book (#39257)
openshift/openshift-docs: registry/index.adoc

[id="registry-overview"]
= {product-title} registry overview
include::modules/common-attributes.adoc[]
:context: registry-overview

toc::[]

> {product-title} can build images from your source code, deploy them, and manage their lifecycle. It provides an internal, integrated container image registry to enable this that can be deployed in your {product-title} environment to locally manage images. This overview contains reference information and links for registries commonly used with {product-title}, with a focus on the internal image registry.

Reviewer: The phrase "to enable this" seems unnecessary. Consider removing.

Author: acked

[id="registry-overview"]
= {product-title} registry overview
include::modules/common-attributes.adoc[]
:context: registry-overview

toc::[]

{product-title} can build images from your source code, deploy them, and manage
their lifecycle. To enable this, {product-title} provides an internal,
integrated container image registry that can be deployed in your {product-title}

> environment to locally manage images. The _Registry Overview_ contains reference information and links for registries commonly used with OpenShift. It focuses on internal Image Registry.

Reviewer 2: Question about "The _Registry Overview_ contains reference information":
Is that referring to this same assembly? If so, perhaps "This overview contains reference information" instead?

Author: Yes. It is referring to this same assembly. Addressed the suggestion.

> environment to locally manage images. The _Registry Overview_ contains reference information and links for registries commonly used with OpenShift. It focuses on internal Image Registry.

Reviewer 2: ```suggestion
environment to locally manage images. The _Registry Overview_ contains reference information and links for registries commonly used with {product-title}, with a focus on the internal image registry.
```
Though that is also a lot of instances of `{product-title}` in this paragraph. Maybe there is a way to rephrase to omit a couple.

Author: acked

> environment to locally manage images. The _Registry Overview_ contains reference information and links for registries commonly used with OpenShift. It focuses on internal Image Registry.

Reviewer 2: Should probably go ahead and unwrap these lines

Author: acked
